class.jetpack.php

```diff
@@ -4023,7 +4023,11 @@ function build_connect_url( $raw = false, $redirect = false, $from = false ) {
 			    'admin.php?page=jetpack-settings' ), $url );
 			}
 		} else {
-			require_once JETPACK__GLOTPRESS_LOCALES_PATH;
+			if ( defined( 'JETPACK__GLOTPRESS_LOCALES_PATH' ) && file_exists( JETPACK__GLOTPRESS_LOCALES_PATH ) ) {
```

Comment:
@iandunn some concerns about this are why `file_exists` instead of `is_file && is_readable`, but also, what happens if the require fails? Maybe it can be written like this? 
```php
if ( defined( 'JETPACK__GLOTPRESS_LOCALES_PATH' ) && require_once JETPACK__GLOTPRESS_LOCALES_PATH ) {
	$gp_locale = GP_Locales::by_field( 'wp_locale', get_locale() );
}
```